Userland/Libraries/LibGfx/Painter.cpp

```diff
@@ -352,6 +352,10 @@ void Painter::fill_rounded_corner(const IntRect& a_rect, int radius, Color color
     if (translated_a_rect.y() < rect.y())
         clip_offset = rect.y() - translated_a_rect.y();
 
+    radius *= scale();
+    rect = rect * scale();
```

Comment:
```suggestion
    rect *= scale();
```

Reply:
I thought I had tried that before and it failed to compile. But now it works. :man_shrugging: